packages/cli-kit/src/api/graphql/extension_specifications.ts

```diff
@@ -37,13 +37,15 @@ export interface RemoteSpecification {
   }
   features?: {
     argo?: {
-      surface: string
+      surface: Surface
     }
   }
 }
 
+type Surface = 'admin' | 'checkout' | 'post-checkout' | 'pos' | 'customer-accounts' | 'all' | null | undefined
```

Comment:
I think these values are not aligned with what comes from the Specifications API 🤔 
Just checked and it returns `customer_accounts` and `point_of_sale` 
Also for post_purchase, the returned surface is `checkout`

Where do these came from?

Reply:
saying this because we defaulted to use the values from the API, not the hardcoded values anymore.

Reply:
yep, right now we are not fetching the values from Core during `dev`, but i'm working on it, so maybe is worth changing the values and merging this PR first